src/components/Card.js

```diff
@@ -1,8 +1,22 @@
-import React from "react";
+import React, {useState} from "react";
+import DeleteSong from "./DeleteSong";
+import UpdateSong from "./UpdateSong";
 
 export const Card = ({ song }) => {
-    
+    const [isDeleteModalVisible, setDeleteModal] = useState(false);
+    const [isUpdateModalVisible, setUpdateModal] = useState(false);
```

Comment:
This works, you can also look at the variable naming so that they match intent
```suggestion
    const [isDeleteModalVisible, setDeleteModalVisibility] = useState(false);
    const [isUpdateModalVisible, setUpdateModalVisibility] = useState(false);
```